src/addrman.h

```diff
@@ -181,14 +182,17 @@ friend class CAddrManTest;
     mutable RecursiveMutex cs;
 
 private:
+    typedef std::unordered_map<int, CAddrInfo> mapInfoType;
+    typedef std::unordered_map<CNetAddr, int, CNetAddrHash> mapAddrType;
```

Comment:
nit: `using` and capitalize class

Reply:
Done: https://github.com/bitcoin/bitcoin/pull/18722/commits/0c3becc83f2cd53ffb5495430d27380f6eda248a#diff-0681caa5b164f2de0cf944039f84b19aR184-R185